_includes/footer.html

```diff
@@ -3,21 +3,27 @@
     <div>
       <img src="/assets/images/logo-gsa.png" alt="GSA logo">
     </div>
-    <div itemscope itemtype="http://schema.org/ContactPoint">
-      <h1>Contact Us</h1>
-      <a href="mailto:18f@gsa.gov" itemprop="email"><i class="fa fa-envelope"></i> 18F@gsa.gov</a>
+    <div>
+      <h1>Work</h1>
+      <a href="/consulting">Consulting</a>
```

Comment:
The <a> tags should have `title` tags for accessibility. They should describe what the link is liking to, in the case, "consulting" would probably be fine.